### DISCUSSION TITLE
Colab GPU free version limits?

### DISCUSSION THREAD

**Discussion by Abdurrafey-Siddiqui** [openai/whisper#560] [Q&A] [answered]
Hi just wanted to know, how often does the Colab GPU reset? Is it every time I connect to the runtime again? Since I've been running whisper on it to transcribe maybe max 5 videos in everytime I connect to the runtime, I'm just wondering for how long I can keep using colab for free.

**Comment by Abdurrafey-Siddiqui** [comment]
the vids are around 3 minutes long each, should I be worried about having to pay afterwards?

**Comment by jongwook** [comment, marked as answer]
From [Google Colab FAQ](https://research.google.com/colaboratory/faq.html#idle-timeouts):

> Colab prioritizes interactive compute. Runtimes will time out if you are idle.
> 
> In the version of Colab that is free of charge notebooks can run for at most 12 hours, depending on availability and your usage patterns. Colab Pro, Pro+, and Pay As You Go offer you increased compute availability based on your compute unit balance.
> 
> In general, notebooks can run for at most 12 hours, depending on availability and your usage patterns. You can expect to experience backend termination if you exhaust your available compute units on a Pro, Pro+, or Pay As You Go plan.

So it seems that 15 minutes of transcription should run just fine with the free tier of Colab.

**Reply by SayakaChiai** [reply]
What's the definition of idle? If you run something for 5 hours and don't use that browser window, would that be considered idle for 5 hours?
